Test/Hspec/Megaparsec.hs

```diff
@@ -176,7 +199,7 @@ initialState s = State
 
 -- | Expectation that argument is result of a failed parser.
 
-shouldFail :: Show a
+shouldFail :: (HasCallStack, Show a)
```

Comment:
Are you sure we need these on non-public helpers? I think it'd be enough to be able to see call stack up to closest public helper, and the internals shouldn't be visible.

Reply:
OK, thanks for the clarification!